src/options.rs

```diff
@@ -133,6 +133,10 @@ pub struct BuildOptions {
     /// allows you to opt out of it.
     #[structopt(long)]
     pub no_cfg_fuzzing: bool,
+
+    #[structopt(long)]
+    /// Don't build with the `sanitizer-coverage-trace-compares` LLVM argument
```

Comment:
```suggestion
    /// Don't build with the `sanitizer-coverage-trace-compares` LLVM argument
    ///
    ///  Using this may improve fuzzer throughput at the cost of worse coverage accuracy.
    /// It also allows older CPUs lacking the `popcnt` instruction to use `cargo-fuzz`;
    /// the `*-trace-compares` instrumentation assumes that the instruction is
    /// available.
```